docs/_static/network/v1.yaml

```diff
@@ -69,4 +69,20 @@ paths:
             application/octet-stream:
               example:
         "404":
-          description: "Transaction (or payload) wasn't found"
\ No newline at end of file
+          description: "Transaction (or payload) wasn't found"
+  /api/graph:
+    get:
+      summary: "Visualizes the DAG as a graph"
+      description: >
+        Walks the DAG as subscribers of the DAG do, rendering it as graph. By default it renders in Graphviz format,
+        which can be rendered to an image using `dot`.
+      operationId: "renderGraph"
+      tags:
+        - transactions
+      responses:
```

Comment:
add errors responses. May reference to generic file. See crypto api as example

Reply:
idem dito